modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/BulkActions.js

```diff
@@ -96,6 +100,18 @@ function BulkActions({
 
 			setCurrentSidePanelActionPayload(sidePanelActionPayload);
 		}
+		else if (
+			onBulkActionItemClick &&
+			typeof onBulkActionItemClick === 'function'
```

Comment:
If we are using `typeof` check, we don't need a falsy check as well. `typeof` covers all falsy values. But, `typeof` is a very strong check. Normally, we would have only a falsy check here, because it will be common that `onBulkActionItemClick` in unset. Is there a reason to think that `onBulkActionItemClick` may be set, but have a different type?

Reply:
I was just being cautious here, I'm glad if just a falsy check works fine, I'll change it